### PR TITLE
[Path] Update heidenhain_post.py

### PR DESCRIPTION
**Pull request by Zibibbo84**
fixed pol_Angle

### CODE REVIEW

**Comment by luzpaz** [comment]
The commit should also be squashed, amended, and rebased.

**Comment by luzpaz** [comment]
Context: https://github.com/FreeCAD/FreeCAD/commit/6e03ba19b7f2b31c6021460dffefed72ebb7d182#r47018295

File: src/Mod/Path/PathScripts/post/heidenhain_post.py [hunk 1]
```diff
@@ -800,7 +800,7 @@ def HEIDEN_PolarArc(pol_cc_X, pol_cc_Y, pol_X, pol_Y, pol_Z, pol_Axis, pol_Incr)
         pol_Angle = 90 - pol_Angle
     # with Y axis zero have the Z+ direction
     elif pol_Axis == "Y":
-        pol_Angle = pol_Angle
+        pol_Angle = 0 + pol_Angle
```
Comment (hyarion) [inline src/Mod/Path/PathScripts/post/heidenhain_post.py:803]: why not use `pass` instead? https://docs.python.org/3/tutorial/controlflow.html#pass-statements
```suggestion
        pass
```

**Review by hyarion — CHANGES_REQUESTED** [review]

File: src/Mod/Path/PathScripts/post/heidenhain_post.py (hunk 1 above)
Comment (Zibibbo84) [reply src/Mod/Path/PathScripts/post/heidenhain_post.py:803]: I'm not a programmer, i know only simple code.
I think your suggestion could work well

**Review by hyarion — APPROVED** [review]

**Comment by luzpaz** [comment]
@sliptonic please merge at your convenience. thanks

**Comment by sliptonic** [comment]
Sorry.  I was waiting for the requested squash/rebase.  It seemed silly to have five commits for two lines of change.

**Comment by sliptonic** [comment]
I can't squash rebase directly.  The PR author has to since it's on his repo branch.

**Comment by sliptonic** [comment]
Zibibbo84  Why don't you close this PR.  I'll cherry pick the important commits and include them with another PR I have open.

**Comment by Zibibbo84** [comment]
Sorry for silly things

**Comment by sliptonic** [comment]
No worries.  https://github.com/FreeCAD/FreeCAD/pull/4497
It'll still maintain you as the author of the commits.

**Pull request closed without merging**